app/components/traffic-measure-template.ts

```diff
@@ -14,10 +14,10 @@ type Args = {
 export default class TrafficMeasureTemplateComponent extends Component<Args> {
   @tracked template?: SafeString;
 
-  @action
-  async didInsert() {
-    await this.fetchTemplate.perform(this.args.concept);
-  }
+  runFetch = modifier(() => {
```

Comment:
This works well, but it might be a bit cleaner to join both the modifier and task into a single `trackedFunction`.